src/Serokell/Data/Variant/Serialization.hs

```diff
@@ -33,7 +32,7 @@ import qualified Data.HashMap.Strict as HM
 --    result type depends on sign (negative ⇒ Int, otherwise UInt).
 
 varMapToObject :: VarMap -> Aeson.Object
-varMapToObject = HM.fromList . map (bimap show' Aeson.toJSON) . HM.toList
+varMapToObject = HM.fromList . map (bimap show Aeson.toJSON) . HM.toList
```

Comment:
Actually, `show` and `show'` from `serokell-util` are `pretty` and `prettyL` from `universum`:

* https://github.com/serokell/universum/blob/b0fc7c1b1ba0aa9c1efba6819d2c7ab0aeea3aae/src/Universum/String/Conversion.hs#L179